src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Bulldozer(),
+                new Excavator(),
+                new Truck()
+        };
 
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
```

Comment:
```suggestion
        for (int i = 0; i < machines.length; i++) {
            Machine machine = machines[i];
            machine .doWork();
            machine .stopWork();
```

Reply:
why do this?
We create a new machine every time - it takes up more memory, doesn't it?